### ISSUE TITLE
Error with model retraining

### ISSUE THREAD

**Issue by GurvanR** [pasquini-dario/LLMmap#7]
Hello Dario,

I want to make a classification task with your method with a new training of the model (on a new dataset). For the moment, as a sanity check I wanted to reproduce results with (1) your model and your dataset and (2) your dataset with a retrained model.

Here is how it went:
- For (1) I ran `python test_model.py ./data/pretrained_models/default -k 1` and obtained an average accuracy of **91% which is aligned** with col (B) of Table 2 of the paper. When looking at model-level accuracy (cf `results_table_1.md`), it seems though quite different, but after all, it is a one time training and not a 10 time as in the paper. 
- For (2) I ran `python train.py ./confs/default.json default_reproduction `  and then `python setup_templates.py data/pretrained_models/default_reproduction` and finally  `python test_model.py ./data/pretrained_models/default_reproduction -k 1` and obtained **2% of accuracy** (cf `results_table_2.md`). 

Do you have any idea of why it could have gone so bad for (2) ?

Thank you very much, again, for your attention and your time.

[results_table_1.md](https://github.com/user-attachments/files/23877843/results_table_1.md)
[results_table_2.md](https://github.com/user-attachments/files/23877842/results_table_2.md)

**Comment by GurvanR** [comment]
I did the commands a second time and it worked!  There might have been an error with the training that had not finished 🤔 .. Sorry for the issue...
